### PR TITLE
docs: add logo

### PR DESCRIPTION
**Pull request by fncolon**
### Pull Request Checklist

- [ ] Have you added new tests to prevent regressions?
- [ ] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [x] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Did you update the typescript typings accordingly (if applicable)?
- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [x] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md)?

### Description Of Change

Add in `README.md` and fix logo in current manual `index.md`
Capture

### Todos

- [x] Fix logo in manual `index.md`
- [x] Show logo in `README.md`

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
-<div>
-  <div class="center logo">
-    ![logo](manual/asset/logo-small.png)
```
Comment (WikiRik) [inline docs/index.md:3]: Why the change from div to p? Wouldn't replacing `manual/asset/logo-small.png` with `docs/images/logo-small.png` be enough?

But even with the other changes, I would recommend making it an internal link so it's not tied to the main branch. If you get what I mean

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+<p align="center">
+  <img src="https://github.com/sequelize/sequelize/blob/main/docs/images/logo.png" />
```
Comment (WikiRik) [inline README.md:2]: Here the same, so please make this an internal reference to the file so when you go back to another tag or commit you use the logo from that commit instead of the main

**Review by WikiRik — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (fncolon) [reply docs/index.md:3]: There's no specific reason i changed to p, but it's my common approach to center something in markdown

Comment (fncolon) [reply docs/index.md:3]: Changed to internal link

File: README.md (hunk 2 above)
Comment (fncolon) [reply README.md]: Changed to internal link

**Review by WikiRik — APPROVED** [review]

File: docs/index.md [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
-  </div>
-  <div class="center sequelize">Sequelize</div>
-</div>
+<p align="center">
+  <img src="images/logo.png" />
```
Comment (sdepold) [inline docs/index.md:2]: Should we go with the small logo here?

**Comment by sdepold** [comment]
I took the liberty and changed the docs myself:
Bildschirmfoto 2021-11-24 um 19 25 43

**Comment by sdepold** [comment]
Centered the name and the log
Bildschirmfoto 2021-11-24 um 19 38 56
o

**Review by WikiRik — APPROVED** [review]
LGTM
